addons/website_forum/views/website_forum.xml

```diff
@@ -46,8 +46,111 @@
             <div t-field="forum.welcome_message"/>
         </div>
 
-        <div class="container mt16 mb32 o_wforum_navbar_container">
-            <div class="navbar navbar-expand-md navbar-light">
+        <div class="container mt16 mb32 ">
```

Comment:
Why this big "seems to be a code move" ?

Reply:
I had to make the nav header a separate template to call it in here but also in forum_user_profile_main template.